src/serial.c

```diff
@@ -539,7 +543,7 @@ static void evaluateCommand(void)
             serialize16(cfg.fw_cruise_throttle);
             serialize16(cfg.fw_idle_throttle);
             serialize16(cfg.fw_scaler_throttle);
-            serialize32(cfg.fw_roll_comp);
+            serialize32(cfg.fw_roll_comp); // Float is Not compatible with Gui. Changing to uint8_t
             serialize8(cfg.fw_rth_alt);

```

Comment:
the comment doesn't match with what's being done. you're serializing uint32, not uint8.